### PR TITLE
feat: Fetch translation authors - Meeds-io/MIPs#121

### PR DESCRIPTION
**Pull request by sergeByishimo**
Use Crowdin GraphQL API to fetch all translation authors to reward once their translations are approved

### CODE REVIEW

File: gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionApprovedTriggerPlugin.java [hunk 1]
```diff
@@ -34,20 +32,45 @@ public void initData() {
...
+        if (object == null) {
+            return eventList;
+        }
+
+        Map<String, Object> translationAuthorsMap = (Map<String, Object>) object;
+        Map<Long, String> translationsAuthors = (Map<Long, String>) translationAuthorsMap.get("translationIdUsernameMap");
+        String translationId = extractSubItem(payload, getPayloadObjectName(), "id");
+        if (translationId != null) {
+            String authorUsername = translationsAuthors.get(Long.parseLong(translationId));
+            LOG.info("authorUsername: "  + authorUsername);
```
Comment (plamarque) [inline gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionApprovedTriggerPlugin.java]: log.debug() maybe ?

Comment (plamarque) [reply gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionApprovedTriggerPlugin.java]: this map will grow in memory, in order to keep an eye maybe we could log.debug its size instead ?

Comment (sergeByishimo) [reply gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionApprovedTriggerPlugin.java:61]: Should have removed it. Not necessary.

Comment (sergeByishimo) [reply gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionApprovedTriggerPlugin.java]: > log.debug() maybe ?

Should have removed it! Not necessary!



> this map will grow in memory, in order to keep an eye maybe we could log.debug its size instead ?

Sure. Do you want to keep an eye on it?

**Review by plamarque — APPROVED** [review]
LGTM. minor comment about logging efficiently

**Pull request merged**